driver/compile.ml

```diff
@@ -75,18 +76,22 @@ let implementation ppf sourcefile outputprefix =
     end else begin
       let bytecode =
         (typedtree, coercion)
-        ++ Translmod.transl_implementation modulename
+        ++ Timings.(time (Transl sourcefile))
+            (Translmod.transl_implementation modulename)
```

Comment:
it would make sense to integrate the `print_if` line below inside the `time` call, as you did below.